mobile_resources_commit

```diff
@@ -1 +1 @@
-6476f5d54c5397507237160a9b911c2f30136fa6
+9aac1a2d0bed58c6b243bec7c160200cf4b46d4a
```

Comment:
just to double check was this intentional for this pr?

Reply:
Yes because it relates to the env update.